WebClient/src/App.vue

```diff
@@ -1,60 +1,37 @@
 <template>
-  <div id="app">
-    <img src="./assets/logo.png">
-    <h1>{{ msg }}</h1>
-    <h2>Essential Links</h2>
-    <ul>
-      <li><a href="https://vuejs.org" target="_blank">Core Docs</a></li>
-      <li><a href="https://forum.vuejs.org" target="_blank">Forum</a></li>
-      <li><a href="https://chat.vuejs.org" target="_blank">Community Chat</a></li>
-      <li><a href="https://twitter.com/vuejs" target="_blank">Twitter</a></li>
-    </ul>
-    <h2>Ecosystem</h2>
-    <ul>
-      <li><a href="http://router.vuejs.org/" target="_blank">vue-router</a></li>
-      <li><a href="http://vuex.vuejs.org/" target="_blank">vuex</a></li>
-      <li><a href="http://vue-loader.vuejs.org/" target="_blank">vue-loader</a></li>
-      <li><a href="https://github.com/vuejs/awesome-vue" target="_blank">awesome-vue</a></li>
-    </ul>
+  <div>
+    <b-navbar sticky toggleable="md" type="dark" variant="info">
+      <b-navbar-toggle target="nav_collapse"></b-navbar-toggle>
+      <b-navbar-brand>Smart Planner</b-navbar-brand>
+      <b-navbar-nav class="ml-auto">
+        <b-nav-item-dropdown v-bind:text="lang" right>
+          <b-dropdown-item v-on:click="lang='pl'">pl</b-dropdown-item>
```

Comment:
I would have changed the dropdown language menu because now it's barely visible. I would make font white and maybe instead of using shortcuts of languages I would use their's full titles.

Reply:
It's only a language dropdown. It shouldn't be as visible as the rest of the website. I would wait with this change for the rest of the website to appear and then decide. What do you think?